pkg/cmd/taskrun/cancel_test.go

```diff
@@ -0,0 +1,107 @@
+package taskrun
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/tektoncd/cli/pkg/test"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+	"github.com/tektoncd/pipeline/pkg/reconciler/v1alpha1/pipelinerun/resources"
+	pipelinetest "github.com/tektoncd/pipeline/test"
+	tb "github.com/tektoncd/pipeline/test/builder"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	k8stest "k8s.io/client-go/testing"
+	"knative.dev/pkg/apis"
+)
+
+func TestTaskRunCancel(t *testing.T) {
+	seeds := make([]pipelinetest.Clients, 0)
+	failures := make([]pipelinetest.Clients, 0)
+
+	for i := 0; i < 1; i++ {
```

Comment:
wonders what we are trying to archive with a loop? :thinking: 

I can see it iterates for once if so we can assign the values directly

Reply:
Thank you for your review!!
It takes increasing test cases into account for future.
(for needing more test datas)

but for now as you mentioned it is no need these test cases :sweat_smile:
I'll delete this loop for avoiding confuse, thanks!!

Reply:
I fixed it !!